hls4ml/templates/vivado/nnet_utils/nnet_pooling.h

```diff
@@ -114,32 +95,19 @@ void pooling1d_cl(data_T data[CONFIG_T::n_in * CONFIG_T::n_filt], res_T res[CONF
     for (int ff = 0; ff < CONFIG_T::n_filt; ff++) {
         // Loop over input image x in steps of stride
         for (int ii = 0; ii < padded_width; ii += CONFIG_T::stride_width) {
+            unsigned overlap_pixel = 0;
             data_T pool[CONFIG_T::pool_width];
             #pragma HLS ARRAY_PARTITION variable=pool complete dim=0
-            // Keep track of number of pixels in image vs padding region
-            unsigned img_overlap = 0;
-            // Loop over pool window x
-            for (int jj = 0; jj < CONFIG_T::stride_width; jj++) {
-                if (ii + jj < CONFIG_T::pad_left || ii + jj >= (padded_width - CONFIG_T::pad_right)) {
-                    // Add padding
-                    pool[jj] = pad_val<data_T, CONFIG_T::pool_op>();
-                    if (CONFIG_T::count_pad)
```

Comment:
It seems like the new code ignores the count_pad option. Is it fine to get rid of it?

Reply:
This was introduced to ensure compatibility with PyTorch. Not sure if it can be removed without breaking bit-perfect agreement with pytorch

Reply:
The count_pad option seems to also be ignored in general in streaming pooling.

Reply:
Actually, padding is not currently supported in streaming, so it's fine that it's ignored.